src/tools/std20/views/test/subrange.test.cpp

```diff
@@ -0,0 +1,192 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+
+// what we are testing
+#include "tools/std20/views.hpp"
+
+// other includes
+#include <forward_list>
+#include <list>
+#include <vector>
+#include "tools/std20/algorithm.hpp"
+
+// convenience typedefs
+//using namespace njoy::tools;
+namespace std20 = nano;
+
+SCENARIO( "subrange" ) {
+
+  const std::vector< int > equal = { -2, -1, 0, 1, 2 };
+  const std::vector< int > modified = { -2, 0, 2, 4, 6 };
+
+  GIVEN( "a container with forward iterators" ) {
+
+    std::forward_list< int > values = { -2, -1, 0, 1, 2 };
+
+    WHEN( "when iterators are used" ) {
+
+      using Range = std20::ranges::subrange< std::forward_list< int >::iterator,
+                                             std::forward_list< int >::iterator,
+                                             std20::ranges::subrange_kind::sized >;
+
+      auto size = std::distance( values.begin(), values.end() );
+      Range chunk( values.begin(), values.end(),size );
+
+      THEN( "the subrange satisfies the required consepcts" ) {
```

Comment:
spelling "concepts", indentation 

Likewise for subsequent THENs within this test